spec/commands/get_pull_request_reviews_info_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe GetPullRequestReviewsInfo do
+  let(:github_user) { create(:github_user) }
+  let(:monkeyci) { create(:github_user, login: 'monkeyci') }
+  let(:pull_request) { create(:pull_request, owner: github_user) }
+
+  def perform(*_args)
+    described_class.for(*_args)
+  end
+
+  context "without any reviews" do
```

Comment:
me parece que el contexto en general lo intentamos partir con un `"when ..."` es un poco más descriptivo. Sería algo como `"when the pull request does not have reviews"` por ejemplo. Sino me tengo que preguntar qué cosa no tiene reviews (a pesar de que se entiende) me parece bueno ser sobre descriptivo ...